internal/accounts/create.go

```diff
@@ -221,21 +211,22 @@ func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow
 		var link string
 		switch selectedNetwork {
 		case config.DefaultTestnetNetwork():
-			log.Info(fmt.Sprintf("%s We will be creating the account on testnet faucet website (%s), which has been prefilled with generated keys", output.WarningEmoji(), util.TestnetFaucetHost))
-			log.Info("\nPlease follow the steps: \n 1. Fill in the captcha, \n 2. Click on 'Create Account' button.\n")
-			output.NextStepPrompt()
+			log.Info("Please complete the following steps in the browser:")
+			log.Info("\n 1. Complete the captcha challenge. \n 2. Click the 'Create Account' button.\n 3. Return to this window.\n")
 			link = util.TestnetFaucetURL(key.PublicKey().String(), crypto.ECDSA_P256)
+
 		case config.DefaultMainnetNetwork():
-			log.Info(fmt.Sprintf("%s We will be creating the account on Flow Port website (%s), which has been prefilled with generated keys", output.WarningEmoji(), util.FlowPortUrl))
-			log.Info("\nPlease follow the steps: \n 1. Click on 'Submit' button, \n 2. Connect existing Blocto or create a new account first, \n 3. Click on confirm, \n 4. Click on approve. \n")
-			output.NextStepPrompt()
+			log.Info("Please complete the following steps in the browser:")
+			log.Info("\n 1. Click on 'Submit' button. \n 2. Connect existing Blocto account or create new.\n 3. Click on confirm and approve transaction.\n")
 			link = util.MainnetFlowPortURL(key.PublicKey().String())
 		}
-		log.StartProgress("Waiting for an account to be created, please finish all the steps in the browser...\n")
-		time.Sleep(time.Second * 2)
+
+		output.ConfirmOpenBrowser()
+
+		log.StartProgress("Waiting for your account to be created, please finish all the steps in the browser...\n")
 		err := util.OpenBrowserWindow(link)
 		if err != nil {
-			return nil, err
+			log.Info(fmt.Sprintf("Could not open a browser window, please navigate to: %s manually", link))
 		}
```

Comment:
Do we know with certainty that this will return an error if the browser can't open? I'm worried about a case where the browser won't open (i.e. due to OS permissions) and this fails silently.